packages/design-system/src/components/tooltip/index.js

```diff
@@ -134,7 +134,7 @@ function Tooltip({
   const positionPlacement = useCallback(
     ({ offset }) => {
       // check to see if there's an overlap with the window edge
-      const neededVerticalSpace = offset.y + offset.height;
+      const neededVerticalSpace = offset.bottom;
```

Comment:
I realized while fixing the dropdown, that the way I was calculating height for tooltip was incorrect. It didn't account for the content of the tooltip being more than one line.